docs/integrations/builtin/request-builtin-node.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Request a built-in node
+pageType: howto
+description: How to request and submit built-in nodes for n8n. 
+---
+
+# Request a built-in node
+
+## Make a feature request
+
+The best way to request a built-in node is to post it in the n8n Community Forum. You can upvote an existing post or create a new post in the [Feature Request > Nodes category](https://community.n8n.io/c/feature-requests/nodes/){:target=_blank .external-link}.
+
+## Submit a built-in node
+
+If you've created a node you'd like to add as an official node in n8n, open a pull request on [GitHub](https://github.com/n8n-io/n8n){:target=_blank .external-link}. An n8n team member will review it. Note that new nodes take time to review and this may take some time. 
+
+## Publish a community node
+
+For a faster way to make your node available, create a community node and publish it on npm. Refer to [Building community nodes](/integrations/community-nodes/build-community-nodes/) for details. Note that these nodes are available [only for self-hosted users and not for n8n Cloud users](/choose-n8n/). 
```

Comment:
We could possibly add a bit to let users know that we do check community nodes and the anonymous telemetry to help us work out if a node should be added to the main set. When this happens we will reach out to the creator via email or as a github issue.